content/docs/1.17/operator.md

```diff
@@ -1013,13 +1013,48 @@ metadata:
+
+```yaml
+        # .Spec.Template.Spec.Containers[0].Args
+        args: ["start", "--log-level=debug"]
+```
+
+Note that tracing and logging at debug level can be both enabled at the same time.
+
 {{< info >}}
-Currently the Operator Lifecycle Manager (OLM) does not offer a way to configure an operator. As a result, if you install the Jaeger Operator via OLM, collecting traces from the Operator is not supported.
+Currently the Operator Lifecycle Manager (OLM) does not offer a way to configure an operator. As a result, if you install the Jaeger Operator via OLM, collecting traces or changing the log level from the Operator is not supported.
 {{< /info >}}
 
 # Monitoring the operator
 
-The Jaeger Operator starts a Prometheus-compatible endpoint on `0.0.0.0:8383/metrics` with internal metrics that can be used to monitor the process.
+The Jaeger Operator starts a Prometheus-compatible endpoint on `0.0.0.0:8383/metrics` with internal metrics that can be used to monitor the process. Interesting metrics to watch are:
+
+```
+# Total number of reconciliations and their outcomes (cumulative)
+controller_runtime_reconcile_total{controller="jaeger-controller",result="error"}
+controller_runtime_reconcile_total{controller="jaeger-controller",result="success"}
+
+# Total number of retries (cumulative)
+workqueue_retries_total{name="jaeger-controller"}
+
+# Current number of reconciliation loops in progress (gauge)
+workqueue_depth{name="jaeger-controller"}
+
+# How many seconds of work has done that is in progress and hasn’t been observed by work_duration. (gauge)
+workqueue_unfinished_work_seconds{name="jaeger-controller"} 0
+
+# How long in seconds processing an item from workqueue takes. (bucket)
+workqueue_work_duration_seconds_bucket{name="jaeger-controller"}
+```
+
+A low number of reconciliation errors is normal, as there might be several processes changing resources at the same time for different reasons. Whenever there's a failure, the operator will attempt to reconcile again, increasing the `workqueue_retries_total` metric. However, if the rate of errors over time keeps increasing, or is beyond a reasonable threshold, an investigation might be required. The reasonable threshold might differ from cluster to cluster depending on what's happening in it, but 10% is a good starting point.
+
+As of v1.17.0, the Jaeger Operator will trigger a reconciliation loop only when the custom resource has changed or when the previous reconciliation has failed. Therefore, it's normal to have this metric with the same value over a long period of time: it only indicates that the custom resource hasn't changed. If this number keeps changing every second, it's indicative that something in the cluster is periodically changing the custom resource, or that the Jaeger Operator is undoing a change that is being done by a different component. Future versions of the Jaeger Operator might trigger a periodic reconciliation loop.
```

Comment:
"it's normal to have this metric with the same value..." - which metric is this referring to? Might be better to be explicit.

Reply:
Changed the doc to be explicit about the other metrics as well.